safe/definitions/field_groups/__init__.py

```diff
@@ -0,0 +1,4 @@
+# coding=utf-8
+
+"""Wrapper for all field groups."""
+from field_groups import *
```

Comment:
__[flake8]__
* [F403] 'from field_groups import *' used; unable to detect undefined names
* [F401] 'field_groups.*' imported but unused

<!-- Comment by [SideCI](https://sideci.com) -->